woocommerce-services.php

```diff
@@ -8,6 +8,7 @@
  * Text Domain: woocommerce-services
  * Domain Path: /i18n/languages/
  * Version: 1.21.0
+ * Requires PHP: 5.3
```

Comment:
This isn't needed, the WordPress 5.2 logic reads the `Requires PHP` header from both the `readme.txt` file and the plugin file. The wordpress.org plugin directory only parses the `readme.txt` file though, so if we're putting it in just one place it should be there.

Unless you prefer to put the `Requires PHP` header in as many places as possible so people have more chances to see it, which I'm fine with :)